modules/actions/handlers/vote_proposals.go

```
@@ -0,0 +1,166 @@
+package handlers
+
+import (
+	auctiontypes "github.com/ODIN-PROTOCOL/odin-core/x/auction/types"
+	coinswaptypes "github.com/ODIN-PROTOCOL/odin-core/x/coinswap/types"
+	minttypes "github.com/ODIN-PROTOCOL/odin-core/x/mint/types"
+	oracletypes "github.com/ODIN-PROTOCOL/odin-core/x/oracle/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	distributiontypes "github.com/cosmos/cosmos-sdk/x/distribution/types"
+	mint1 "github.com/cosmos/cosmos-sdk/x/mint/types"
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+	slashingtypes "github.com/cosmos/cosmos-sdk/x/slashing/types"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+	ibctypes "github.com/cosmos/ibc-go/v2/modules/apps/transfer/types"
+	ibc1 "github.com/cosmos/ibc-go/v2/modules/core/02-client/types"
+	"github.com/forbole/bdjuno/v3/modules/actions/types"
+	"reflect"
+	"strings"
+)
+
+type Params struct {
+	Key       string
+	ValueType string
+}
+
+type ModuleParamsInfo struct {
+	ModuleName string
+	ParamPairs paramtypes.ParamSetPairs
+}
+
+func GetVoteProposals(ctx *types.Context, payload *types.Payload) (interface{}, error) {
```

error is never returned